app/views/searches/_success_saved.html.haml

```diff
@@ -1 +1 @@
-= link_to("search saved! Click to show all your saved searches", searches_user_path, class: "btn btn-success")
\ No newline at end of file
+= link_to("search saved! Click to show all your saved searches", searches_user_path, class: "btn btn-success")
```

Comment:
jeśli nie przewidujesz innych wersji językowych to może zostać, jeśli zaś byś chciała wprowadzić inne wersje językowe to proponuje zaczać używać w nowych featureach I18n a tam gdzie jest już hardkodowany tekst to przy okazji refactoru pozmieniać ;)